docs/CONTRIBUTING.adoc

```diff
@@ -78,17 +78,17 @@ To create a Polkadot companion PR:
 . Pull latest Polkadot master (or clone it, if you haven't yet).
 . Override your local cargo config to point to your local substrate (pointing to your WIP branch): place `paths = ["path/to/substrate"]` in `~/.cargo/config`.
 . Make the changes required and build polkadot locally.
-. Submit all this as a PR against the Polkadot Repo. Link to your Polkadot PR in the _description_ of your Substrate PR as "polkadot companion: [URL]"
+. Submit all this as a PR against the Polkadot Repo.
+. Link to your Polkadot PR in the _description_ of your _Substrate_ PR as "polkadot companion: [URL]"
+. Link to your Substrate PR in the _description_ of your _Polkadot_ PR as "Companion PR for [URL]"
 . Now you should see that the `check_polkadot` CI job will build your Substrate PR agains the mentioned Polkadot branch in your PR description.
-. Wait for reviews on both the Substrate and the Polkadot pull request.
-. Once the Substrate pull request runs green, a member of the `parity` github group can comment on the Substrate pull request with `bot merge` which will:
-    - Merge the Substrate pull request.
-    - In case the pull request origins from https://github.com/paritytech/polkadot directly and not from a fork:
-        - The bot will push a commit to the Polkadot pull request updating its Substrate reference.
-        - The bot will merge the Polkadot pull request once all its CI checks are green.
-    - In case the pull request origins from a fork (relevant for pull requests by external contributors):
-        - You need to push a commit to the Polkadot pull request updating the Substrate reference.
-        - You need to merge by commenting `bot merge` on the Polkadot pull request once all CI checks on the pull request are green.
+. Someone will need to approve the Polkadot PR before the Substrate CI will go green. (The Polkadot CI failing can be ignored as long as the polkadot job in the _substrate_ PR is green).
+. Wait for reviews on both the Substrate and the Polkadot PRs.
+. Once the Substrate PR runs green, a member of the `parity` github group can comment on the Substrate PR with `bot merge` which will:
+    - Merge the Substrate PR.
+    - The bot will push a commit to the Polkadot PR updating its Substrate reference.
+    - If the polkadot PR origins from a fork then a project member may need to press `approve run` on the polkadot PR.
```

Comment:
I think there is another caveat, that bot doesn't work with forks on org accounts, instead of individual ones, e.g. https://github.com/paritytech/polkadot/pull/3477 cc @joao-paulo-parity

Reply:
That's right, see https://github.com/isaacs/github/issues/1681#issuecomment-825453532 for context

Reply:
I don't know where is the button "approve run" but I guess it is correct.